MantidQt/MantidWidgets/src/MantidWSIndexDialog.cpp

```diff
@@ -108,15 +107,15 @@ QMultiMap<QString, std::set<int>> MantidWSIndexWidget::getPlots() const {
  * @returns True if waterfall plot selected
  */
 bool MantidWSIndexWidget::isWaterfallPlotSelected() const {
-  return m_waterfallOpt ? m_waterfallOpt->isChecked() : false;
+  return (m_plotOptions->currentText() == "Waterfall Plot");
```

Comment:
I think tr("Waterfall Plot") = "Waterfall Plot". If you doubt this, I suggest testing on a non-Windows machine.